### ISSUE TITLE
On spread + favor effect for the Nurgle "Fever" symptom doesn't work when spreading to armies (MechyFox)

### ISSUE THREAD

**Issue by Autumnchain** [chadvandy/cbfm_wh3#872]
Currently, in v 2.4.0, the "Fever" symptom for Ku'Gath's plagues does not work if the plague is spread to an army (but it does work if a settlement is infected with the plague). This does not reflect the description text of the effect.

This mod (https://steamcommunity.com/sharedfiles/filedetails/?id=2941416579) fixes the issue.

**Comment by MechyFoxVT** [comment]
I'm the author of the mod shown above. To fix the issue, I went into the faction_set_effect_bundles table in the database and changed a couple cells to match the effects found on the Necrotising Flesh symptom's cells.

Here's a screenshot of the cells I changed to fix Fever. It seems like the issue was just a bad copy/paste job.
Before Fix: 
![image](https://user-images.githubusercontent.com/35346582/222792916-2d72e968-9418-462c-ade1-2de438eaa747.png)

After Fix:
![image](https://user-images.githubusercontent.com/35346582/222793097-2bce0592-7de6-4b13-b3c6-eb9b4fdbb8c9.png)
